services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -547,6 +550,120 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
         }
     }
 
+    @Ignore("Encapsulation type is not implemented yet in APIv2")
+    @Unroll
+    @Tags(HARDWARE)
+    def "System allows to create/update encapsulation type for a flow\
+(#encapsulationCreate.toString() -> #encapsulationUpdate.toString())"() {
+        given: "Two active not neighboring switches"
+        def noviflowSwitchesIds = topology.activeSwitches.findAll { !it.centec && it.ofVersion == "OF_13" }*.dpId
```

Comment:
`!it.centec` is not the same as `it.noviflow`. Well, in our particular topology it is, but still.

Reply:
fixed